### PR TITLE
Added gpu flags to table

### PR DESCRIPTION
**Pull request by joseph-cooke**

### CODE REVIEW

**Comment by arsalanakhter** [comment]
I believe there are two kinds of flags for a100. One is a100 and the other is a100@80g. Not sure if we need to mention that.

**Review by arsalanakhter — APPROVED** [review]

**Comment by joseph-cooke** [comment]
There is actually only one flag for the a100. The general flag will get you either a 40g or 80g card but you can restrict it to the 80g by using the --constraint=a100@80g with the normal flag. Thank you!!

**Comment by arsalanakhter** [comment]
Oh that clarifies. Thank you!

**Review by arsalanakhter — APPROVED** [review]

File: docs/source/hardware/hardware_overview.md [hunk 1]
```diff
@@ -143,4 +152,20 @@ widths: auto
   - 3(*x*8)
 :::
 
+:::{note}
+The a100 GPUs come in two variants, 41 GB and 82 GB memory versions. If your job requires the 82 GB version, please include the constraint flag in your job:
```
Comment (arsalanakhter) [inline docs/source/hardware/hardware_overview.md:156]: I'd change the text to the following:

The a100 GPUs come in two variants, 41 GB and 82 GB memory versions. If your job requires the 82 GB version, please include the `--constraint=` flag in your job:

for consistency with the language above in cpu section

File: docs/source/hardware/hardware_overview.md [hunk 2]
```diff
@@ -143,4 +152,20 @@ widths: auto
...
+
+`srun -p gpu --gres=gpu:a100:1 --constraint=a100@80 --pty /bin/bash`
+
+and for sbatch
+
+`#SBATCH -p gpu`
+
+`#SBATCH --gres=gpu:a100:1`
+
+`#SBATCH --constraint=a100@80g`
```
Comment (arsalanakhter) [inline docs/source/hardware/hardware_overview.md:168]: Can we make a code block? something like

```
#SBATCH -p gpu
#SBATCH --gres=gpu:a100:1
#SBATCH --constraint=a100@80g
```

File: docs/source/hardware/hardware_overview.md (hunk 1 above)
Comment (joseph-cooke) [reply docs/source/hardware/hardware_overview.md]: Absolutely and done!!

**Comment by arsalanakhter** [comment]
Perfect. Thank you!

**Review by arsalanakhter — APPROVED** [review]

**Comment by joseph-cooke** [comment]
Thank you!!

**Pull request merged**